db/monitor/GSM5ACD.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<monitor name="LG LCD Monitor-19M38HB">
+	<controls>
```

Comment:
Please, add `<caps ...` as the first tag, as it is in other monitors. Source this information from #78:

```
Raw output: (prot(monitor)type(lcd)model(MP76)cmds(01 02 03 0C E3 F3)vcp(02030405080B0C101214(01 05 06 07 08 0B)1516181A4D4E4F5260(01 03 04)626C6E70878DACAEB6C0C6C8C9D6(01 04)DFE0E1E3(00 01 02 03 04 10 11 12 13 14)ECEFFD(00 01)FE(00 01 02)F4F5F6F7F8F9FF)mswhql(1)mccs_ver(2.1))
```

Reply:
Was done in https://github.com/ddccontrol/ddccontrol-db/pull/77/commits/70191fc71deea749ded24cf37e26a3515da822aa